api/src/main/java/jakarta/config/ConfigProvider.java

```diff
@@ -70,7 +70,7 @@ public static Config getConfig() {
      *            the Classloader used to register the configuration instance
      * @return the configuration instance for the given class loader
      */
-    public static Config getConfig(ClassLoader cl) {
+    public static Config getConfig(ClassLoader loader) {
         return ConfigProviderResolver.instance().getConfig(cl);
```

Comment:
Shouldn't `getConfig(cl);` be updated as well?

Reply:
Fixed, build failed... I just committed the suggestion from Emily and missed this.